ExternalProjects/LibBizHash/README.md

```diff
@@ -0,0 +1,21 @@
+LibBizHash is the unmanaged side for BizHawk's hashing.
+
+CRC32 code is taken from [zlib-ng](https://github.com/zlib-ng/zlib-ng) with massive slashing of code and various tweaks. This code is licensed under the zlib license.
+SHA1 is code is taken from [SHA-Intrinsics](https://github.com/noloader/SHA-Intrinsics) with some tweaks. This code is under the public domain.
+All modifications by the BizHawk team are licensed under MIT.

```

Comment:
nitpick: "modifications" are not licensed, the modified copy of the source is. More importantly this should include build instructions.